pinot-tools/src/main/java/org/apache/pinot/tools/segment/converter/DictionaryToRawIndexConverter.java

```diff
@@ -262,20 +268,28 @@ private void updateMetadata(File segmentDir, String[] columns, String tableName)
     properties.save();
   }
 
+  @Override
```

Comment:
why need to override those methods here?

Reply:
b/c picocli requires Command to implement callable (that is defined in `org.apache.pinot.tools.Command` interface. 

Technically speaking we no longer need the `org.apache.pinot.tools.Command` interface because picocli provide the exact same thing. I will add a TODO to the follow up task to clean this up.

Reply:
but this actually is a good catch. we no longer need to first parse argument call converter.convert(). I can simply do this in execute(). let me make the change.